bali/events/utils.py

```diff
@@ -0,0 +1,16 @@
+from kombu import Connection
```

Comment:
`utils.py` only contains `dispatch` method now, and `dispatch` is a very important part to trigger events. Why not just naming it `dispatch.py`?